packages/compass-import-export/src/csv/csv-utils.spec.ts

```diff
@@ -756,7 +756,9 @@ describe('parseCSVValue', function () {
         '{"$ref":"namespace","$id":{"$oid":"642af890571e13e609f9069a"}}',
         'ejson'
       )
-    ).to.deep.equal(new DBRef('namespace', new ObjectId()));
+    ).to.deep.equal(
+      new DBRef('namespace', new ObjectId('642af890571e13e609f9069a'))
+    );
```

Comment:
I don't have the time right now to dig into "how did this test _ever_ pass" but it seems like a fun one :)